File: go/Movie.go
package movie_rental

type Movie struct {
	_title     string
	_priceCode uint
}

func (m Movie) getPriceCode() uint {
	return m._priceCode
}

func (m Movie) getTitle() string {
	return m._title
}

const REGULAR = 0
const NEW_RELEASE = 1
const CHILDRENS = 2

File: php/src/Movie.php
<?php
namespace Kata;

class Movie
{
    const REGULAR = 0;
    const NEW_RELEASE = 1;
    const CHILDRENS = 2;
    private mixed $_priceCode;
    private mixed $_title;

    public function __construct($title, $priceCode)
    {
        $this->_title = $title;
        $this->_priceCode = $priceCode;
    }

    public function getPriceCode()
    {
        return $this->_priceCode;
    }

    public function setPriceCode(int $arg)
    {
        $this->_priceCode = $arg;
    }

    public function getTitle()
    {
        return $this->_title;
    }

}